src/deploy/functions/prepare.ts

```diff
@@ -138,6 +138,13 @@ export async function prepare(
   // Setup environment variables on each function.
   for (const endpoint of backend.allEndpoints(wantBackend)) {
     endpoint.environmentVariables = wantBackend.environmentVariables;
+    if (endpoint.platform === "gcfv2") {
```

Comment:
nit: Should we also check whether it's actually an event-handling function, or just set it unilaterally in gen2? I'm probably fine either way.

Reply:
We should only be doing it if `backend.isEventTriggered`, otherwise I believe we can force HTTPS functions into parsing as event functions.